docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 # requirements file for documentation
 sphinx
-sphinx_rtd_theme
 sphinx-argparse
+# sphinx_rtd_theme
+# guzzle_sphinx_theme
```

Comment:
Maybe link the [page with the alternatives](https://www.writethedocs.org/guide/tools/sphinx-themes/) instead?
```suggestion
# https://www.writethedocs.org/guide/tools/sphinx-themes/
```

Reply:
These lines shouldn't appear. :smile: I just tried some news themes to distinguish visually from the semver2 line. I've tried some, but didn't like them that much, so I've adapted the Alabaster theme.

So I guess, we don't really need that line (unless you think the current layout is horribly broken). :wink:

Reply:
Just get rid of them. I just thought these comments were there intentionally 😜

Reply:
Haha, not at all! :laughing: After writing so many lines, it was just a leftover. 😜